backend/data_validation/base_data_val.py

```diff
@@ -29,25 +32,79 @@ def __init__(self, path_to_config: str = "config.yaml"):
             f"../data/{self.config['year']}{self.config['event_code']}_match_data.json",
         )
 
-        # Retrieves match schedule
-        try:
-            with open(
-                self.config.get("path_to_match_schedule", "../data/match_schedule.json")
-            ) as file:
-                self.match_schedule = load(file)
-        except FileNotFoundError:  # We want to ignore if it doesn't exist because get_match_schedule() will create it.
-            pass
+        self._event_key = str(self.config["year"]) + self.config["event_code"]
+        # determines both if were using tba for match shedule and whether we're running tba checks
```

Comment:
```suggestion
        # Determines both if were using tba for match shedule and whether we're running tba checks
```
Just a nitpick to keep comment style consistent with other files.